client/src/components/Navbar.jsx

```diff
@@ -0,0 +1,104 @@
+import { useRef, useEffect } from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import NavAccess from "../services/NavAccess";
+
+function Navbar({ handleChange, navOpen, navVisible }) {
+  const menuRef = useRef(null);
+
+  useEffect(() => {
+    if (menuRef.current) {
+      const navAccessKeyboard = new NavAccess(menuRef);
+      navAccessKeyboard.addEventListener();
+
+      return () => {
+        navAccessKeyboard.removeEventListener();
+      };
+    }
+    return () => {};
+  }, [menuRef]);
+
+  return (
+    <>
+      <button
+        type="button"
+        className="block absolute left-4 top-4 md:hidden"
+        onClick={handleChange}
+        aria-label="Ouvrir le menu"
+      >
+        <img src="/src/assets/images/menuBurger.svg" alt="icone menu burger" />
```

Comment:
Is alt information necessary for navigation when there's an aria-label attribute? It would be preferable to use this button, which is an image, to transmit information via the alt attribute and a title attribute, or to put this image in css.